src/collections/_documentation/development/sdk-dev/event-payloads/sdk.md

```diff
@@ -25,28 +28,36 @@ capture and transmit the event.
 
 : _Optional_. A list of packages that were installed as part of this SDK or the
   activated integrations. Each package consists of a `name` in the format
-  `source:identifier` and a semver `version`. If the source is a git repository,
-  a checkout link and git reference (branch, tag or SHA) should be used.
+  `source:identifier` and `version`.  
+  If the source is a Git repository, the source should be `git`, the
+  `identifier` should be a checkout link and the `version` should be a Git
+  reference (branch, tag or SHA).
```

Comment:
I tried to update this to the best of my understanding. Please check if it is accurate.